src/textual/widgets/_markdown.py

```diff
@@ -589,7 +589,7 @@ def __init__(self, href: str) -> None:
 
     async def on_mount(self) -> None:
         if self._markdown is not None:
-            await self.update(self._markdown)
+            self.update(self._markdown)
 
     async def load(self, path: Path) -> bool:
```

Comment:
Make method non-async?

```suggestion
    def load(self, path: Path) -> bool:
```